packages/remark-lint/src/rules/required-metadata.mjs

```diff
@@ -0,0 +1,38 @@
+import { lintRule } from 'unified-lint-rule';
+import { visit } from 'unist-util-visit';
+
+const requiredKeys = ['introduced_in', 'llm_description'];
+const checks = requiredKeys.map(key => new RegExp(`<!--\\s*${key}=.*?-->`));
+
+const hasRequiredMetadata = (tree, vfile) => {
```

Comment:
Can you clean up this a bit? I assume this function will replace the rule on doc-kit?